### DISCUSSION TITLE
How can I use summarize(总结) function?

### DISCUSSION THREAD

**Discussion by 2p990i9hpral** [GaiZhenbiao/ChuanhuChatGPT#880] [Q&A] [answered]
How can I use “Summary”? I think this feature would be useful, but it doesn't work!
When I click the "总结" button, nothing happens.

**Comment by Keldos-Li** [comment, marked as answer]
Its full name may be "Summarize Index File (Button)" 🤔 Then you can understand why……

**Reply by 2p990i9hpral** [reply]
Thanks, I thought that button was a method to minimize token usage by summarizing previous dialog history. 
Many other AI clients provide this function. How about implementing this? I think it's quite simple.

**Reply by GaiZhenbiao** [reply]
I'll consider that
